package.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "electric-boray",
-    "description": "buckets boray proxy service",
+    "name": "buckets-mdplacement",
+    "description": "buckets metadata placement service",
     "version": "1.0.4",
```

Comment:
A ver bump would be good... perhaps to 2.x

Reply:
:+1:

Reply:
Done